nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,18 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nrf_modem 2.6.1
+***************
+
+Sockets
+=======
+
+* Fixed a bug where the :c:func:`nrf_recv` and :c:func:`nrf_recvfrom` functions erroneously returned ``-1`` and set ``errno`` to :c:macro:`NRF_EAGAIN` instead of returning ``0`` when these three conditions were met:
```

Comment:
```suggestion
* Fixed a bug where the :c:func:`nrf_recv` and :c:func:`nrf_recvfrom` functions erroneously returned ``-1`` and set ``errno`` to ``NRF_EAGAIN`` instead of returning ``0`` when these three conditions were met:
```